example_project/articles/migrations/0002_custom_generic_permissions.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 2.1.14 on 2019-11-14 10:41
```

Comment:
would be great if you could try this with django 2.2+

Reply:
I would expect Django 2.1 support to break and TravisCI to fail.

Generating on 2.2 would make the example project migrations dependent `auth.0011_update_proxy_permissions` (Django >= 2.2) rather than `auth.0009_alter_user_last_name_max_length` (Django ~2.1).

Reply:
Also note the CI tests do  test Django 2.2.

Reply:
@auvipy If it is ok with you, I would like to resolve this conversation.

Reply:
Resolving conversation since requiring Django 2.2 would fail the current configuration of Travis CI